docs/sources/reference/components/prometheus/prometheus.operator.podmonitors.md

```diff
@@ -34,6 +34,7 @@ Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `forward_to` | `list(MetricsReceiver)` | List of receivers to send scraped metrics to. | | yes
 `namespaces` | `list(string)` | List of namespaces to search for PodMonitor resources. If not specified, all namespaces will be searched. || no
+`informer_sync_timeout` | `duration` | Timeout for initial sync of PodMonitor resources. | `1m` | no
```

Comment:
If it's for the initial sync, should it be called `initial_sync_timeout`? Is "informer" a k8s term?

Reply:
I think it's best to leave it as "informer", because the error also mentions the word "informer":

> timeout exceeded while configuring informers

By naming it `informer_sync_timeout`, there's a bigger chance that the users will realise that they need to tweak `informer_sync_timeout`.